PendingReleaseNotes.md

```diff
@@ -21,6 +21,9 @@ an example usage
 - `NodeAffinity` can be applied to `rook-ceph-agent DaemonSet` with `AGENT_NODE_AFFINITY` environment variable.
 - `NodeAffinity` can be applied to `rook-discover DaemonSet` with `DISCOVER_AGENT_NODE_AFFINITY` environment variable.
 - Rook does not create an initial CRUSH map anymore and let Ceph do it normally
+- Flexvolume plugin now support dynamic PVC resizing.
```

Comment:
Could we also get something in the docs that indicates how to resize a volume? For example, maybe a brief section added to the [block storage](/Documentation/ceph-block.md) topic?